com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -395,6 +400,47 @@ class Job(_ResourceElement):
         >>> print (jobs[0].health)
         healthy
     """
+    def get_application_trace(self, path=None, name=None):
```

Comment:
It's a little inconsistent that the method is `get_application_trace`, the docstring says `application log` and the file name suffix is `app_logs.tar`.

Reply:
We should be consistent within this definition and consistent with existing Streams doc, e.g.:

https://www.ibm.com/support/knowledgecenter/en/SSCRJU_4.2.1/com.ibm.streams.jmxapi.doc/api/com/ibm/streams/management/job/JobMXBean.html#retrieveApplicationLogAndTraceFiles-java.lang.String-

Reply:
Fixed. Should all be "application logs" now.

Reply:
I'm fine with the naming, but the doc strings is not consistent with the JMX description, specifically: 
_ all application log and trace files for the job in tar archive format._

ie. no mention of trace or what format the resultant file is.

Reply:
> Returns an HTTPS URI to retrieve all application log and trace files for the job in tar archive format.

Okay, to be consistent with the JMX doc I added two pieces of information:
* The trace information is also retrieved 
* The retrieved logs are from the associated job
The doc had already mentioned it's saved as a tar file.